### DISCUSSION TITLE
BlueRetro with Atari 2600/7800

### DISCUSSION THREAD

**Discussion by Colrut** [darthcloud/BlueRetro#583] [Q&A]
Good morning

Curious if there is a way to get this to work with Atari consoles.  While have not tried this at home, the wired Sega controllers are apparently compatible with the DB9 connectors on Atari consoles.  Would this (see link below) allow wireless controllers to work with Atari consoles?  Does anyone have experience with this?

https://www.blue-retro.com/products/cable-ss/

Thank you

**Comment by darthcloud** [comment]
I do not support that website product in anyway. They are using the project name as their brand which violate the project license.

As the project main page say you can use BlueRetro on Atari system using the Parallel 2p firmware and cable:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#parallel-2p-5v-6-buttons-each-adapter-cable

**Reply by Colrut** [reply]
Thank you.  I had no idea that they were doing this.  Is there a recommended place to purchase the core and pre-assembled cables?  I am located in USA.

**Comment by Sowden** [comment]
Hey there dude.  So I have been on quite a bit of a journey to solve this question for myself.  There is a dude over on Tindie that is selling a product for the Atari systems, but it's a bit of leg work to get there.  Now I'm not a electrician, so someone correct me if I'm wrong.  So things like N64, Gamecube, PS1/2, etc use a 3.3V electronics to power their controllers.  That's the reason why you can just solder some wires together and just make it work.  You can find that product here.

https://www.tindie.com/products/grechtech/gtrr-modular-blueretro/

But for things like Genesis, Atari systems need some more, like 5V in order to make it work.  So they sell a addon pack that all it takes is plugging in some jumpers and the use of a screw driver.  You can find that here.

https://www.tindie.com/products/grechtech/gtrr-levelshifter-add-on-kits/

Both of them are in stock, as of this writing.  And they fully support darthcloud here, unlike that link you posted.  And do business with them, they are really good people.  I am currently waiting for my two pieces to come in the mail.  They are coming from the UK and one is on its way, but the other is stuck in Royal Mail.  I hadn't heard this, but Royal Mail right now is having a nasty Russian ransomware problem, yikes!  So I'm not sure when I will get my piece.  However I started a thread posted below about this, because I want to get it working with my ST and get the mouse working with it.  I will be posting about this further there, so let's continue this discussion on the post below, cause I'd like to hear your progress.  But anyway, hope that might give you a head start.  

https://github.com/darthcloud/BlueRetro/issues/401#event-7414488876

**Reply by GrechTech** [reply]
Thanks for recommending us!

The post situation here is slowly sorting itself out here, hopefully there's not too much delay for your items from here on!

Now full parcel shipping has resumed here we can also finally start selling these as complete units, with pre-built options available for users who want to avoid the work in wiring & setup:

https://www.tindie.com/products/grechtech/gtrr-levelshifter-modular-5v-blueretro/

Thanks!
